app/src/main/java/org/hackillinois/android/view/MentorAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package org.hackillinois.android.view
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.mentor_recycleview.view.*
+import org.hackillinois.android.R
+import org.hackillinois.android.model.mentor.MentorModel
+
+class MentorAdapter internal constructor(private val mentorList: ArrayList<MentorModel>) : RecyclerView.Adapter<MentorAdapter.ViewHolder>() {
```

Comment:
Can you switch this to just a List<MentorModel>? It's a little cleaner to not depend on the actual list implementation.a